ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/BonsaiReferenceTestWorldState.java

```diff
@@ -87,7 +89,7 @@ public ReferenceTestWorldState copy() {
    */
   @Override
   protected void verifyWorldStateRoot(final Hash calculatedStateRoot, final BlockHeader header) {
-    if (!header.getStateRoot().equals(Hash.ZERO)) {
+    if (!disableRootHashVerification) {
```

Comment:
~~If we are going to do it this way, we should probably also set disableRootHashVerification in the retesteth subcommand.  I don't think it is being used currently, but we shouldn't leave it broken~~

Nevermind - it looks like retesteth subcommand is still using forest